src/main.rs

```diff
@@ -234,18 +271,16 @@ async fn handle_outcomes(
     let mut succeeded_receipt_ids: Vec<String> = vec![];
 
     for outcome in outcomes.values() {
-        match outcome.outcome.status {
-            near_indexer::near_primitives::views::ExecutionStatusView::Failure(_) => {
-                failed_receipt_ids.push(outcome.id.to_string())
-            }
-            near_indexer::near_primitives::views::ExecutionStatusView::SuccessReceiptId(_)
-            | near_indexer::near_primitives::views::ExecutionStatusView::SuccessValue(_) => {
+        let status: db::enums::ExecutionStatus = outcome.outcome.status.clone().into();
+        match status {
+            db::enums::ExecutionStatus::Success => {
                 succeeded_receipt_ids.push(outcome.id.to_string());
             }
-            near_indexer::near_primitives::views::ExecutionStatusView::Unknown => {
-                error!(
+            db::enums::ExecutionStatus::Failed => failed_receipt_ids.push(outcome.id.to_string()),
+            _ => {
```

Comment:
It is an impossible state, there are only two variants in the enum.

Reply:
The enum has 3 variants, I'll change `_` to explicit `Pending` one, but we don't expect it.